spec/requests/carts/show_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe "Cart" do
+  
+  describe "show cart" do
```

Comment:
remove this Describe and rename the other one as this one, same with the other tests